pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -189,8 +195,10 @@ stdenv.mkDerivation rec {
     "-DUSE_APPLE_ICU=OFF"
     "-DUSE_OPENGL_OR_ES=OFF"
     "-DUSE_SYSTEM_MALLOC=ON"
-  ] ++ lib.optionals (!stdenv.isLinux) [
+  ] ++ lib.optionals (!systemdSupport) [
     "-DUSE_SYSTEMD=OFF"
+  ] ++ lib.optionals (!dbusSupport) [
+    "-DDBUS_PROXY_EXECUTABLE=true"    # webkitgtk has a false bubblewrap->dbus dependency; this fixes it
```

Comment:
I do not think this is a false dependency. The sandbox will [try to bind it](https://github.com/WebKit/WebKit/blob/2dbb11247d999671af1c8bd14837fcf4961e1c53/Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp#L682) and fail :

```ShellSession
$ nix shell -f ~/Projects/nixpkgs bubblewrap -c bwrap --ro-bind true true hello
bwrap: Can't find source path true: No such file or directory
```

And even if we replace it with something like `${coreutils}/bin/true`, that will not be on `PATH` in the sandbox, likely rendering webkit unusable.

Reply:
Thank you for the review.  I will need to investigate more closely to figure out how to explain to webkitgtk that if it is built without dbus support, it has no need for a dbus-proxy.  Apparently it does not know this.

In the meantime, I have revised this patch to deal only with systemdSupport.  I will submit dbusSupport as a separate patch.